src/core_modules/capture-core/components/WidgetEnrollment/dataMutation/dataMutation.ts

```diff
@@ -1,39 +1,37 @@
-// @flow
-import { useDataMutation, type QueryRefetchFunction } from '@dhis2/app-runtime';
+import { useDataMutation } from '@dhis2/app-runtime';
 import { useRef } from 'react';
+import type { Mutation } from 'capture-core-utils/types/app-runtime';
 
-const enrollmentUpdate = {
+const enrollmentUpdate: Mutation = {
     resource: 'tracker?async=false&importStrategy=UPDATE',
     type: 'create',
-    data: enrollment => ({
+    data: (enrollment: any) => ({
         enrollments: [enrollment],
     }),
 };
 
-const enrollmentDelete = {
+const enrollmentDelete: Mutation = {
     resource: 'tracker?async=false&importStrategy=DELETE',
     type: 'create',
-    data: enrollment => ({
+    data: (enrollment: any) => ({
         enrollments: [enrollment],
     }),
 };
 
-const processErrorReports = (error) => {
-    // $FlowFixMe[prop-missing]
+const processErrorReports = (error: any): string => {
     const errorReports = error?.details?.validationReport?.errorReports;
     return errorReports?.length > 0
-        ? errorReports.reduce((acc, errorReport) => `${acc} ${errorReport.message}`, '')
+        ? errorReports.reduce((acc: string, errorReport: any) => `${acc} ${errorReport.message}`, '')
         : error.message;
 };
 
-
 export const useUpdateEnrollment = (
-    refetchEnrollment: QueryRefetchFunction,
-    refetchTEI: QueryRefetchFunction,
-    onError?: ?(message: string) => void,
-    onSuccess?: ({redirect?: boolean}) => void,
+    refetchEnrollment: any,
```

Comment:
import and use QueryRefetchFunction. Do this every time you encounter the QueryRefetchFunction type

```
import type { QueryRefetchFunction } from 'capture-core-utils/types/app-runtime';
refetchEnrollment: QueryRefetchFunction
```